marlowe-runtime/marlowe-follower/Options.hs

```diff
@@ -0,0 +1,51 @@
+module Options where
+
+import Language.Marlowe.Runtime.Core.Api (ContractId (..), parseContractId)
+import Network.Socket (HostName, PortNumber)
+import Options.Applicative (argument, auto, execParser, fullDesc, header, help, helper, info, long, maybeReader,
+                            metavar, option, progDesc, short, strOption, value)
+
+data Options = Options
+  { port       :: PortNumber
+  , queryPort  :: PortNumber
+  , host       :: HostName
+  , contractId :: ContractId
+  }
+
+getOptions :: IO Options
+getOptions = execParser $ info (helper <*> parser) infoMod
+  where
+    parser = Options <$> portParser <*> queryPortParser <*> hostParser <*> contractIdParser
+
+    portParser = option auto $ mconcat
+      [ long "port-number"
+      , value 3715
+      , metavar "PORT_NUMBER"
+      , help "The port number of the chain seek server. Default value: 3715"
```

Comment:
Thanks for including default values in the help! :smile:

Reply:
I think there is actually an `optparse-applicative` combinator that adds this. I'll look into this, it would be nice to have this generated instead.